tests/ltp/patches/ltp_listxattr_listxattr02_fix.patch

```diff
@@ -0,0 +1,60 @@
++ Patch Description: Tests were failing as no xattr support in loop device
++ So modofied the tests to use root filesystem with xattr support
++ Issue 297: Sgx enclave is getting aborted while test is trying to access invalid address
++ https://github.com/lsds/sgx-lkl/issues/297
+diff --git a/testcases/kernel/syscalls/listxattr/listxattr02.c b/testcases/kernel/syscalls/listxattr/listxattr02.c
+index 98a0985b3..fd5c50ca0 100644
+--- a/testcases/kernel/syscalls/listxattr/listxattr02.c
++++ b/testcases/kernel/syscalls/listxattr/listxattr02.c
+@@ -36,7 +36,8 @@
+ #define SECURITY_KEY   "security.ltptest"
+ #define VALUE  "test"
+ #define VALUE_SIZE     (sizeof(VALUE) - 1)
+-#define TESTFILE    "testfile"
++#define TESTFILE    "mntpoint/testfile"
++#define MNTPOINT    "mntpoint"
+
+ char longpathname[PATH_MAX + 2];
+
+@@ -47,7 +48,7 @@ static struct test_case {
+ } tc[] = {
+        {TESTFILE, 1, ERANGE},
+        {"", 20, ENOENT},
+-       {(char *)-1, 20, EFAULT},
++//     {(char *)-1, 20, EFAULT}, // TODO: Enable once git issue 297 is fixed
```

Comment:
Can you explain this change? I also don't understand issue #297

Reply:
LTP tests expects EFAULT error returned when trying to access an invalid address, it doesn't return EFAULT but aborts.

Reply:
Ok, thanks. Can you clean up issue #297, so we can properly triage it? I can provide feedback what detail we require (e.g. a gdb trace) to fix this.

Reply:
@prp cleaned https://github.com/lsds/sgx-lkl/issues/297